docs/instrumentation-list.yaml

```diff
@@ -694,11 +684,6 @@ libraries:
     target_versions:
       javaagent:
       - org.apache.axis2:axis2-jaxws:[1.6.0,)
-  - name: jaxws-2.0-cxf-3.0
-    source_path: instrumentation/jaxws/jaxws-2.0-cxf-3.0
-    scope:
-      name: io.opentelemetry.jaxws-2.0-cxf-3.0
-    target_versions: {}
```

Comment:
these three being removed were the result of some temporary directories existing in my local that were being picked up by mistake